alerting/provider/messagebird/messagebird.go

```diff
@@ -0,0 +1,47 @@
+package messagebird
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/TwinProduction/gatus/alerting/provider/custom"
+	"github.com/TwinProduction/gatus/core"
+)
+
+const (
+	restAPIURL = "https://rest.messagebird.com/messages"
+)
+
+// AlertProvider is the configuration necessary for sending an alert using Messagebird
+type AlertProvider struct {
+	AccessKey  string `yaml:"access-key"`
+	Originator string `yaml:"originator"`
+	Recipients string `yaml:"recipients"`
```

Comment:
Is there any reason why we're using `recipients` and not `recipient`?
Based on the documentation, the field `recipients` is an array, but the implementation is a string.

Regardless, I'd rather rename `originator` and `recipients` to `from` and `to` respectively. 
I know it doesn't align with Messagebird, but I just think that `from` and `to` are a lot more intuitive. 

Let me know what you think

Reply:
The reason is, it supports multiple recipients, it could be defined as `31619191919,31619191918` which would allow you to send multiple destinations due to that I would prefer recipients as a naming but if you prefer `from` and `to` over `originator` and `recipients`, I could do required changes. If you ask my opinion, I would prefer to stick with API definitions in naming so that API would be aligned with the messagebird provider and make it easy to add new properties from messagebird API. Let me know your thoughts about it.

Reply:
![image](https://user-images.githubusercontent.com/15699766/100039860-227f0b00-2dd4-11eb-8249-e01b9c201a8d.png)
Looking at the documentation, even they refer recipients and originator as to and from interchangeably 😩 

That said, the former is used more often, so I'll yield

Reply:
> the former is used more often
I thought you meant `from` and `to` fields. Sorry for inconvenience